src/dqe_fun.erl

```diff
@@ -10,9 +10,12 @@
 
 -export([init/0, reg/1, lookup/2, list/0, render/1]).
 -export_type([spec/0]).
+
+-ignore_xref([behaviour_info/1, list/0, lookup/2, reg/1, render/1]).
+
 -define(TBL, dqe_fun).
 
--type return_type() :: histogram | metric.
+-type return_type() :: histogram | metric | time.
```

Comment:
return type in dqe_fun is the **function** return type, that is something entirely different from the operation map you link there.

from the POV of dqe_fun time is an argument, like integer or float (two lines down). A function can't return a time.